frontend/background.js

```diff
@@ -1,12 +1,33 @@
 console.log("background.js is executed when extension is installed or refreshed");
 
-const listOfInjectableURLRegex = [ // change these to the list of allowed news sources
-    "^https:\\/\\/www\\.google",
-    "^https:\\/\\/www\\.youtube\\.com",
-    "^https:\\/\\/www\\.stackoverflow\\.com",
-    "^https:\\/\\/www\\.bbc\\.com\\/news\\/"
+var listOfInjectableURLRegex = [ // change these to the list of allowed news sources
+    // "^https:\\/\\/www\\.google",
+    // "^https:\\/\\/www\\.youtube\\.com",
+    // "^https:\\/\\/www\\.stackoverflow\\.com",
+    // "^https:\\/\\/www\\.bbc\\.com\\/news\\/"
 ]
 
+// Parses allowed news sources from news-sources.json
+function parseAllowedNewsSource() {
+    // Fetches data from local .json file
+    fetch('../static/news-sources.json')
```

Comment:
Is it possible to preserve the promise here?

Since this runs asynchrounosly, it would be nice to `.then` it later to make sure it's complete before we actually do any validation (as the array is empty initially, and is only populated once the promise is resolved)